app/models/place.rb

```diff
@@ -0,0 +1,4 @@
+class Place < ApplicationRecord
+validates :city, presence: true, uniqueness: true
+validates :where, presence: true, length: { minimum: 1, maximum: 300 }
+end
```

Comment:
missind empty line at the end of the file

Reply:
OK